### DISCUSSION TITLE
Project Ideas

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#2642] [Q&A]
Hey people I need some help. As I am in my final year now (Diploma) I have to make a major project. For that I have been searching for project Ideas. You can be as much creative as you want while suggesting the project idea. I will need at least 3-4 project ideas to present in front of my faculties. Really looking forward to your suggestions

**Comment by suitupstan** [comment]
Project Generator 🤔

**Comment by importhuman** [comment]
Are there any particular criteria your project has to meet? Also, how much time will you have to work on it?

**Reply by kaiwalyakoparkar** [reply]
No Just should be worthy of being called a major project (eg: Not a single page website) :)

**Reply by importhuman** [reply]
Any particular time frame you'll have for the project?

**Reply by kaiwalyakoparkar** [reply]
I will have 4 months to build the project

**Reply by importhuman** [reply]
- Social website/application 
- Chat app
- Shopping website/application

**Reply by izhar360** [reply]
> Any particular time frame you'll have for the project? 

What if one has a whole year? I'm talking about me.

**Reply by importhuman** [reply]
The above mentioned ones with more fine tuning/features perhaps? Otherwise think of an application/website you use in your life and see what you'd like to recreate on your own :)

**Comment by siddhant-khisty** [comment]
A RPG game through a discord bot that uses machine learning to tweak the game experience by putting forth random senarios and a fixed number of options to handle the senario.

**Reply by kaiwalyakoparkar** [reply]
Interesting. Thanks for suggestion :heart:

**Comment by ashwinexe** [comment]
A platform which allows final year projects to be showcased and possibly open sourced.

## Reason: 
Most of the final year projects are made for the sake of marks and left inactive after they are graded. A platform which encourages students to showcase their projects would be a nice encouragement to maintain them for long run as open source.

### This would also mean
- beginners get a kickstart in open source as they showcase their final projects.
- Constructive Feedback from community 
- They get to network with like minded folks as they explore others projects

**Reply by izhar360** [reply]
Greattt one! How about a github repo for now?

**Comment by TishaJhabak1014** [comment]
> Hey people I need some help. As I am in my final year now (Diploma) I have to make a major project. For that I have been searching for project Ideas. You can be as much creative as you want while suggesting the project idea. I will need at least 3-4 project ideas to present in front of my faculties. Really looking forward to your suggestions

How about DL model for image segmentation!
